terraform/variables.tf

```diff
@@ -65,18 +65,6 @@ variable "ci_service_account_member" {
   description = "The service account member for deploying revisions to Cloud Run"
 }
 
-variable "github_owner_id" {
```

Comment:
Why did you remove these? Are you planning on not using WIF? Without either and owner or an enterprise id you'd be allowing all traffic which is not great. You want wif to shed requests from things outside of our enterprise.

Reply:
Oh for some reason I thought I didn't need them based on a WIF comment in my previous PR. I'll re-add them.